packages/orm/src/collections/CustomReference.ts

```diff
@@ -1,25 +1,35 @@
-import { Entity, IdOf, LoadHint, Loaded } from "../EntityManager";
-import { AbstractRelationImpl } from "./AbstractRelationImpl";
+import { Entity, IdOf } from "../EntityManager";
 import { ensureNotDeleted, fail, Reference } from "../index";
+import { AbstractRelationImpl } from "./AbstractRelationImpl";
 
-export type CustomReferenceOpts<
-  T extends Entity,
-  U extends Entity,
-  H extends LoadHint<T>,
-  N extends never | undefined
-> = {
+export type CustomReferenceOpts<T extends Entity, U extends Entity, N extends never | undefined> = {
+  // We don't capture the return value of load b/c we purposefully want get to re-calc from `entity`
```

Comment:
👍 comment

Reply:
Yep. :-) I'd had a version for awhile that was capturing the return value and at the "oh fuck, right..." moment thought I'd tell myself not to try that again, because w/o the comment I'm sure I would try it again in about 3-6 months.